knip.ts

```diff
@@ -1,56 +1,35 @@
-import type { KnipConfig } from "knip";
+import type { KnipConfig } from 'knip';
 
 const config: KnipConfig = {
-  ignoreWorkspaces: ["packages/nuxt"],
-  ignoreDependencies: ["cz-conventional-changelog"],
```

Comment:
Add `commitizen` explicitly to activate the plugin and this wouldn't be reported.